README.md

```diff
@@ -93,11 +93,14 @@ const PeerId = require('peer-id')
 
 The key format is detailed in [libp2p-crypto](https://github.com/ipfs/js-libp2p-crypto).
 
-### `create([opts])`
+### `create([opts], cb)`
```

Comment:
As mentioned before, let's keep the top level callback named 'callback', it reads better and we always used until the 'refactor' tornado started pushing `cb` everywhere